pkgx.yaml

```diff
@@ -1,5 +1,4 @@
 dependencies:
-  pkgx.sh/brewkit: ^0
+  pkgx.sh/brewkit: ^0 || ^1
```

Comment:
you like this better than `<2`?

Reply:
What and allow `v−1` 😂. I prefer this, but yeah either would be fine.